components/hscesium/ol-cesium-object-map-item.class.ts

```diff
@@ -0,0 +1,5 @@
+import {DataSource, ImageryLayer} from 'cesium';
+export class OlCesiumObjectMapItem {
```

Comment:
@raitisbe When you neither intend to create new objects of type `OlCesiumObjectMapItem` with the 'new' keyword nor implement its methods, then I think the interface is a cleaner option (it does not compile into JS and is only used for static type-checking).
The same for the `ParamCacheMapItem` class.
Just a note ;-)